src/mrinufft/io/nsp.py

```diff
@@ -207,19 +204,13 @@ def _pop_elements(array, num_elements=1, type=np.float32):
 
 def write_trajectory(
     trajectory: np.ndarray,
-    FOV: tuple[float, ...],
-    img_size: tuple[int, ...],
     grad_filename: str,
-    norm_factor: float = KMAX,
-    gamma: float = Gammas.HYDROGEN,
-    raster_time: float = DEFAULT_RASTER_TIME,
     check_constraints: bool = True,
     TE_pos: float = 0.5,
-    gmax: float = DEFAULT_GMAX,
-    smax: float = DEFAULT_SMAX,
     pregrad: str | None = None,
```

Comment:
These codes are kinda used in legacy codebases (even SPARKLING). I feel its good practice to have both options available for now and add a warning. WDYT?